randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -763,24 +774,31 @@ Extra - asset_id: 27
   * Normal Door to Climb/Door to Crateria Super Missile Room
   > Pickup (Super Missile Expansion)
       All of the following:
-          Speed Booster and Shinespark (Beginner)
-          Ice Beam or Shinespark (Advanced)
+          Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 129
+          Any of the following:
+              Ice Beam
+              # Advanced Shinespark: https://youtu.be/AKO0Zz2MryE
+              Shinespark (Advanced)
 
 > Pickup (Super Missile Expansion); Heals? False
   * Layers: default
   * Pickup 11; Category? Minor
   > Door to Climb (North)
       All of the following:
           Morph Ball
-          Grapple Beam or Space Jump
+          Any of the following:
+              Grapple Beam or Space Jump
+              Damage Boosting (Beginner) and Normal Damage ≥ 121
```

Comment:
does this damage get reduced if you have HJ?

Reply:
I haven't considered it, but the ceiling is pretty low so maybe not. In the bigger picture, a single etank solves the problem in both cases for 99% of seeds.

Reply:
still nice to have something like this, for custom energy tank value cases :)

Reply:
HJ makes it harder because of the low ceiling. Perfect movement is 1 hit which is 60 health from the spikes. 2 hits is easy. I changed it to 1 hit for accuracy.

Reply:
Cool, if HJ makes it harder then ignore